server/sockets/list.js

```diff
@@ -348,6 +384,156 @@ const emitRemoveMemberOnLeaveCohort = socket =>
+      .then(doc => {
+        if (doc) {
+          const {
+            cohortId: { _id: cohortId, memberIds: cohortMemberIds },
+            viewersIds
+          } = doc;
+
+          const list = { ...doc, cohortId };
+
+          if (type === ListType.LIMITED && removedViewers) {
+            removedViewers.forEach(id => {
+              const userId = id.toString();
+              const isCohortMember = checkIfArrayContainsUserId(
+                cohortMemberIds,
+                userId
+              );
+
+              if (listClients.has(userId)) {
+                const { viewId, socketId } = listClients.get(userId);
+
+                if (viewId === listId) {
+                  socket.broadcast
+                    .to(socketId)
+                    .emit(ListActionTypes.LEAVE_ON_TYPE_CHANGE_SUCCESS, {
+                      cohortId,
+                      isCohortMember,
+                      listId,
+                      type
+                    });
+                }
+              }
+
+              if (dashboardClients.has(userId)) {
+                const { socketId } = dashboardClients.get(userId);
+
+                socket.broadcast
+                  .to(socketId)
+                  .emit(ListActionTypes.DELETE_SUCCESS, listId);
+              }
+
+              if (cohortClients.has(userId)) {
+                const { viewId, socketId } = cohortClients.get(userId);
+
+                if (viewId === cohortId.toString()) {
+                  socket.broadcast
+                    .to(socketId)
+                    .emit(ListActionTypes.DELETE_SUCCESS, listId);
+                }
+              }
+            });
+          }
+
+          viewersIds.forEach(id => {
+            const userId = id.toString();
+
+            if (dashboardClients.has(userId)) {
+              const { socketId } = dashboardClients.get(userId);
+
+              socket.broadcast
+                .to(socketId)
+                .emit(ListActionTypes.FETCH_META_DATA_SUCCESS, {
+                  [list._id]: { ...responseWithList(list, userId) }
+                });
+            }
+
+            if (cohortClients.has(userId)) {
+              const { viewId, socketId } = cohortClients.get(userId);
+
+              if (viewId === cohortId.toString()) {
+                socket.broadcast
+                  .to(socketId)
+                  .emit(ListActionTypes.FETCH_META_DATA_SUCCESS, {
+                    [list._id]: { ...responseWithList(list, userId) }
+                  });
+              }
+            }
+          });
+
+          socket.broadcast
+            .to(`sack-${listId}`)
+            .emit(ListActionTypes.CHANGE_TYPE_SUCCESS, data);
+        }
+      });
+  });
+
+const removeListMember = (
+  socket,
+  dashboardClients,
+  listClients,
+  cohortClients
+) =>
+  socket.on(ListActionTypes.REMOVE_MEMBER_SUCCESS, data => {
+    const { listId, userId } = data;
+
+    // Broadcast to that removed user if he is at dashboard view
```

Comment:
remove comments, dashboardClients in use so it's clear that you emit to users on dashbaord